src/locales/en.json

```diff
@@ -561,6 +561,7 @@
         "service": "The connection to your bank is operated and secured by our partner : Linxo. [Learn more](https://support.cozy.io/article/147-linxo)."
       }
     },
+    "maintenance": "maintenance",
```

Comment:
I don't think that this key fits here, in the middle of keys related to real konnectors.
In the `<AccountPicker />` component, we already have a translated value, which key starts with `account_picker`. This locales should go there.

Also, the translation `maintenance` should start with a capital.

If it has not been done yet, we should confirm with @Claiw or @joel-costa that it fits with what they have in mind.